### PR TITLE
deployment/docker/victorialogs/fluentbit-docker/docker-compose.yml: update fluentbit version from v2.1.4 to v3.0.2

### PR DESCRIPTION
**Pull request by denisgolius**
see also https://fluentbit.io/announcements/v3.0.0/

### CODE REVIEW

**Comment by hagen1778** [comment]
Thanks for PR! Have you verified the docker env continues to work correctly after the version change?

File: deployment/docker/victorialogs/fluentbit-docker/docker-compose.yml [hunk 1]
```diff
@@ -2,7 +2,8 @@ version: "3"
 
 services:
   fluentbit:
-    image: cr.fluentbit.io/fluent/fluent-bit:2.1.4
+#    image: cr.fluentbit.io/fluent/fluent-bit:2.1.4
```
Comment (tenmozes) [inline deployment/docker/victorialogs/fluentbit-docker/docker-compose.yml:5]: please remove commented code

**Review by tenmozes — CHANGES_REQUESTED** [review]

**Comment by denisgolius** [comment]
> Thanks for PR! Have you verified the docker env continues to work correctly after the version change?

Yes, I didn't see any problems while running the new version for two days.

**Review by hagen1778 — APPROVED** [review]
LGTM

**Pull request merged**